docs/DEPLOYMENT-VERIFICATION.md

```diff
@@ -0,0 +1,168 @@
+# Test Pilot Test Plan
+
+## ENVIRONMENTS:
```

Comment:
Do you think it's worth mentioning the `Preferences` drop-down in this section? Like, "this _should_ match with the drop-down, buuuut..."